cuda/private/template_helper.bzl

```
@@ -180,12 +180,19 @@ def _generate_toolchain_clang_build(repository_ctx, cuda, clang_path_or_label):
         clang_path = "%{clang_path}",
     )
 
+    cuda_path_for_subst = ""
+    if cuda.path:
+        cuda_path_for_subst = _to_forward_slash(cuda.path)
+    else:
+        # Use repository_ctx.name to get the correct canonical name
+        cuda_path_for_subst = "external/{}/clang_compiler_deps".format(repository_ctx.name)
```

```suggestion
        cuda_path_for_subst = "{}/clang_compiler_deps".format(Label("@cuda//cuda").workspace_root)
```

This is a native speak to get the runtime path within some (generated) repo.